dali/operators/reader/loader/numpy_loader.cc

```diff
@@ -24,16 +24,18 @@
 
 namespace dali {
 TypeInfo TypeFromNumpyStr(const std::string &format) {
```

Comment:
Why not to reuse `TypeFromFormatStr` from https://github.com/NVIDIA/DALI/blob/master/dali/util/pybind.h#L103?